hamocc/mo_hamocc_init.F90

```diff
@@ -87,8 +87,8 @@ subroutine hamocc_init(read_rest,rstfnm_hamocc)
          &            do_sedspinup,sedspin_yr_s,sedspin_yr_e,sedspin_ncyc,                         &
          &            inidic,inialk,inipo4,inioxy,inino3,inisil,inid13c,inid14c,swaclimfile,       &
          &            with_dmsph,pi_ph_file,l_3Dvarsedpor,sedporfile,ocn_co2_type,use_M4AGO,       &
-         &            do_n2onh3_coupled,lkwrbioz_off,lTO2depremin,shelfsea_maskfile,sedqualfile,   &
-         &            ldyn_sed_age
+         &            do_n2o_coupled,lkwrbioz_off,lTO2depremin,shelfsea_maskfile,sedqualfile,      &
+         &            ldyn_sed_age,do_nh3_coupled
```

Comment:
```suggestion
         &            do_n2o_coupled,do_nh3_coupled,lkwrbioz_off,lTO2depremin,shelfsea_maskfile,      &
         &            sedqualfile.ldyn_sed_age,
```

Reply:
maybe nice to keep`n2o` and `nh3` options together also here?

Reply:
Done.